doc/source/user_guide/numpy_images.rst

```diff
@@ -97,27 +96,27 @@ Boolean arithmetic can be used to define more complex masks: ::
 Color images
 ------------
 
-All of the above is true of color images, too: a color image is a
-NumPy array, with an additional trailing dimension for the channels:
+All of the above is true for color images too: a color image is a
+NumPy array with an additional trailing dimension for the channels:
 
     >>> cat = data.chelsea()
     >>> type(cat)
     <type 'numpy.ndarray'>
     >>> cat.shape
     (300, 451, 3)
 
-This shows that ``cat`` is a 300-by-451 pixel image with three
-channels (red, green, and blue).
-As before, we can get and set pixel values:
+This shows that ``cat`` is a 300-by-451 pixel image with three channels
+(in this case, red, green, and blue). As before, we can get and set
```

Comment:
(red, green, blue) is by far the most frequent case, I'm not sure it's worth saying "in this case"